analyses/transcriptomic-dimension-reduction/scripts/generate-multipanel-plot.R

```diff
@@ -40,6 +46,11 @@ if (!dir.exists(plot_directory)) {
   dir.create(plot_directory, recursive = TRUE)
 }
 
+#### Source custom function ---------------------------------------------------
```

Comment:
Why not hard code the script that needs to be sourced? See another example from this module.

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/46cf6ccb119312ccae6122ac94c51710df01f6da/analyses/transcriptomic-dimension-reduction/scripts/run-dimension-reduction.R#L56

Do you see any reason why the input would change?

Reply:
Good question. I do not anticipate the input changing so I'll go ahead and hard code this.